.github/workflows/ci.yml

```
@@ -68,3 +68,37 @@ jobs:
+          ref : main
+          token : ${{secrets.ACTION_TOKEN}}
+          path : home-protector-infra
+
+      - name : Update backend image
+        if : success()
+        run : |
+          cd home-protector-infra/overlays/develop/backend
+          sed -i 's|image: 637423458012.dkr.ecr.ap-northeast-2.amazonaws.com/home-protector.*|image: 637423458012.dkr.ecr.ap-northeast-2.amazonaws.com/home-protector:${{ github.sha }}|' patch-image.yaml
+          cat patch-image.yaml
+
+      - name : Push updated manifest
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)